appium/tests/specs/mock/setup/RespectsDisallowAttesterSearchForDomainsOnPreDomainBasic.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { MockApi } from 'api-mocks/mock';
+import {
+    SplashScreen,
+} from '../../../screenobjects/all-screens';
+import {attesterPublicKeySamples} from "../../../../api-mocks/apis/attester/attester-endpoints";
+import SetupKeyScreen from "../../../screenobjects/setup-key.screen";
+import MailFolderScreen from "../../../screenobjects/mail-folder.screen";
+import NewMessageScreen from "../../../screenobjects/new-message.screen";
+
+describe('SETUP: ', () => {
+
+  it('respects disallow_attester_search_for_domains on a per-domain basis', async () => {
```

Comment:
File should be called `RespectsDisallowAttesterSearchForDomainsOnPerDomainBasic` instead of `RespectsDisallowAttesterSearchForDomainsOnPreDomainBasic` (`per` instead of `pre`) according to test name